### PR TITLE
10.0 25062018

### PR DESCRIPTION
**Pull request by iledarn**

### CODE REVIEW

File: website_multi_company_sale/doc/changelog.rst [hunk 1]
```diff
@@ -1,3 +1,13 @@
+`1.2.0`
+-------
+
+- **ADD:** Users with ``Current Backend Website`` specified have access only to sale orders from their website
```
Comment (yelizariev) [inline website_multi_company_sale/doc/changelog.rst]: May be *... from **that** website* ?

File: website_multi_company_sale/models/sale_order.py [hunk 2]
```diff
@@ -16,10 +16,22 @@ def sale_get_order(self, force_create=False, code=None, update_pricelist=False,
             # company.id seems to be the same as self.id, but let's use variant
             # from original sale_get_order
             self = self.with_context(force_company=company.id)
-        return super(Website, self).sale_get_order(force_create, code, update_pricelist, force_pricelist)
+        sale_order = super(Website, self).sale_get_order(force_create, code, update_pricelist, force_pricelist)
+
+        website_id = self._context.get('website_id')
+        if sale_order and website_id:
+            sale_order.website_id = website_id
```
Comment (yelizariev) [inline website_multi_company_sale/models/sale_order.py]: If sale order already has a website_id that different current one, you must create new order instead.
Or be sure that sale_get_order returns only order of the current website

Comment (yelizariev) [reply website_multi_company_sale/models/sale_order.py]: It must work  with Super User too

File: website_multi_company_sale/doc/changelog.rst [hunk 3]
```diff
@@ -1,3 +1,13 @@
+`1.2.0`
+-------
+
+- **ADD:** Users with ``Current Backend Website`` specified have access only to sale orders from that website
+
+`1.1.1`
+-------
+
+- **Improvement:** group sale orders by website
```
Comment (yelizariev) [inline website_multi_company_sale/doc/changelog.rst]: You don't need to change version twice in the same PR

File: website_multi_company_sale/models/sale_order.py [hunk 4]
```diff
@@ -10,16 +10,28 @@ class Website(models.Model):
...
             # sale_order_id in the session
 
             # company.id seems to be the same as self.id, but let's use variant
             # from original sale_get_order
             self = self.with_context(force_company=company.id)
-        return super(Website, self).sale_get_order(force_create, code, update_pricelist, force_pricelist)
+        sale_order = super(Website, self).sale_get_order(force_create, code, update_pricelist, force_pricelist)
+
+        website_id = self._context.get('website_id')
+        if sale_order and website_id and force_create:
```
Comment (yelizariev) [inline website_multi_company_sale/models/sale_order.py]: Why you check for ``force_create``?

File: website_multi_company_sale/doc/changelog.rst (hunk 3 above)
Comment (iledarn) [reply website_multi_company_sale/doc/changelog.rst]: How to write a comment about two things - one is about group feature and another - about new security rule?

File: website_multi_company_sale/models/sale_order.py (hunk 4 above)
Comment (iledarn) [reply website_multi_company_sale/models/sale_order.py]: sale_get_order is used in different places - from many templates. But I need to update website_id in sale order only when user click ``Add to Cart`` - only in that case force_crate is True

**Comment by yelizariev** [comment]
@iledarn 
```

website_multi_company_sale/tests/test_sale_get_order.py:2:1: F401 'urlparse' imported but unused
website_multi_company_sale/tests/test_sale_get_order.py:19:9: F841 local variable 'demo_user' is assigned to but never used

```

**Comment by iledarn** [comment]
@yelizariev , done lints

**Pull request merged**